.github/actions/delete-deployments/action.yml

```diff
@@ -0,0 +1,56 @@
+name: delete-deployments
+
+description: Deletes the deployments created by a workflow run in the scope of the PR.
+
+inputs:
+  owner:
```

Comment:
I don't see the point of specifying the repo here, except if we plan to call that action from a workflow that  would handle multiple repositories: using GH env variables to find the owner/repo would be better